src/pages/content-edit/nav.css

```diff
@@ -2,7 +2,7 @@
 @value layout_css: "../layout/layout.css";
 @value anchor: #446;
 @value iconTransition: color 0.2s ease-in;
-@value colorRemove from "../../buttons.css";
+@value buttons_css: "../../buttons.css";
```

Comment:
Would you move this up w/ `layout_css` please so they can buddies united in doing the same thing?